HOW/social/process/README.md

```diff
@@ -0,0 +1,3 @@
+# Process Standards
```

Comment:
If this is under `social`, I assume it's about social processes. It's hard for me to imagine a social standard in the context of Holochain as described above anyway, and I would be much less capable of telling the difference between a social standard and a social process.